tests/test_taar_legacy.py

```diff
@@ -0,0 +1,18 @@
+"""Test suite for TAAR Legacy Job."""
+
+import pytest
+import requests
+
+AMO_LEGACY_RECS_URI =\
+    "https://addons-dev.allizom.org/api/v3/addons/replacement-addon/"
+
+
+@pytest.fixture
+def is_api_up():
```

Comment:
Avoid hitting a live server in the testing suite. One way to test `fetch_legacy_replacement_masterlist` would be to mock/monkeypatch `requests.get` with a custom function that reads from a predefined dictionary. You can specify in the test what the schema of the data you're expecting looks like and double check that the execution of the main routine is working as expected.

The doc page for the [pytest monkeypatch fixture](https://docs.pytest.org/en/latest/monkeypatch.html) is pretty good.  Here's [an example](https://github.com/mozilla/python_mozetl/blob/master/tests/test_topline_summary.py#L316-L318) of monkeypatch being used in this repository for tests.

Reply:
Awesome! Thanks for the detailed feedback. PR has been updated to include appropriate test coverage in this manner.